README.md

```diff
@@ -28,12 +28,48 @@ When the reader has completed this Code Pattern, they will understand how to:
 * [Flask](http://flask.pocoo.org/): is a micro web framework.
 * [ngrok](http://ngrok.com/): allows you to open an HTTPS tunnel into your REST API.
 
-## Prerequisites:
+## Featured technologies
+
+* [Artificial Intelligence](https://medium.com/ibm-data-science-experience): Artificial intelligence can be applied to disparate solution spaces to deliver disruptive technologies.
+* [Python](https://www.python.org/): Python is a programming language that lets you work more quickly and integrate your systems more effectively.
+
+## Prerequisites
 
 The following are prerequisites to start developing this application:
 
 * Xcode
 * Turi Create (5.0 Beta 2)
 * Cocoapods
 * Python 2.7 & PIP
+* Flask
 * Jupyter
+* ngrok
+
+# Steps
+
+(NOTE: I still need to add screenshots/pictures to all the steps!)
+
+1. [Download and organize the latest MovieLens dataset](#1-download-and-organize-the-latest-movielens-dataset)
+2. [Train the Turi Create Recommender model](#2-train-the-turi-create-recommender-model)
+3. [Run the backend](#3-run-the-backend)
+4. [Deploy the iOS app](#4-deploy-the-ios-app)
+
+### 1. Download and organize the latest MovieLens dataset
+
+There's a script available in the root directory of this repository, called `setup.sh`. Run this script - it'll download the movielens data, and organize it, for you.
```

Comment:
I don't think you need the commas at the end of the second sentence, and the hyphen could be substituted for something that reads more smoothly, like "Run this script, which will ...".